int8 binFullness
int8 batteryLevel
